docs/design/hub-and-spoke/crud/create/README.md

```diff
@@ -2,23 +2,23 @@
 
 ## Current implementation
 
-There are multiple issues occur and might occur because of the design of crud (create operations in particular):
-- rules might not be installed due to some errors or switches disappearance during installation
-- there is still one last cache where we store flow resources
-- it is almost impossible to track down what went wrong during flow creation
-- it is hard to support current code base without FSM
+There are multiple issues occur and might occur because of the design of CRUD (create operation in particular):
+- rules might not be installed due to some errors or switches disappearance during installation,
+- there is still one last cache where we store flow resources,
+- it is almost impossible to track down what went wrong during flow creation,
+- it is hard to support current code base without FSM.
 
 ![Flow create current version](flow-create-current.png "Flow create current version")
 
 ## Flow creation using hub and spoke
-This approach helps to detect failures during installation and revert changes if something went wrong. Also it allows to
-handle timeout cases and add retries or another custom logic. In scope of this refactoring we should get rid of last
-"cache" in the system, it is ResourceCache where we store cookies, vlans and meters.
+This approach helps to detect failures during installation and revert changes if something went wrong. Additionally,
+it allows to handle timeout cases and add retries or another custom logic. In scope of this refactoring we should get 
+rid of last "cache" in the system, it is ResourceCache where we store cookies, VLAN, and meters.
```

Comment:
We could also write like this:

"In **the** scope of this refactoring, we should get rid of **the** last 'cache' in the system, **which** is ResourceCache where we store cookies, VLAN, and meters."